migrator/src/main/resources/templates/publication.json.ftl

```diff
@@ -27,7 +27,7 @@
     "name" : "publicationsystem:Summary",
     "type" : "STRING",
     "multiple" : false,
-    "values" : [ "Some summary" ]
+    "values" : [ "Publication" ]
```

Comment:
Does this need to be more specific to denote it is a summary value?

Reply:
This is just place holder text for the top level publication (e.g. Dec-2017) as there are no summaries to migrate from the data for publications. This will be changed in the CMS under the 'Summary' header by SMEs so I think it's ok for now if you agree? I was trying to improve it from 'some summary' 🙂